modin/experimental/core/execution/native/implementations/hdk_on_native/hdk_worker.py

```diff
@@ -14,16 +14,21 @@
 """Module provides ``HdkWorker`` class."""
 from typing import Optional, Tuple, List, Union
 
+from packaging import version
+
 import pyarrow as pa
 import os
 
+import pyhdk
 from pyhdk.hdk import HDK, QueryNode, ExecutionResult, RelAlgExecutor
 
 from .base_worker import DbTable, BaseDbWorker
 
 from modin.utils import _inherit_docstrings
 from modin.config import HdkLaunchParameters, OmnisciFragmentSize, HdkFragmentSize
 
+_CAST_DICT = version.parse(getattr(pyhdk, "__version__", "0")) <= version.parse("0.7.0")
```

Comment:
In what situation would `pyhdk` not contain `__version__`?

Reply:
I've added this `getattr` because `doc_checker` fails with `AttributeError: __version__`.

Reply:
It's better to add `__version__` mock here: https://github.com/modin-project/modin/blob/0e0a2e65df895fb69b41efc318d26996ad3e23fe/scripts/doc_checker.py#L529